zerver/lib/onboarding_steps.py

```diff
@@ -52,6 +52,9 @@ def to_dict(self) -> dict[str, str]:
     OneTimeNotice(
         name="interleaved_view_messages_fading",
     ),
+    OneTimeNotice(
+        name="topic_resolved",
```

Comment:
I think we want to call this `intro_resolved_topics_modal`; the previous name is confusing, since it happens before you resolve it...